packages/wdio-allure-reporter/src/index.js

```diff
@@ -120,6 +124,20 @@ class AllureReporter extends WDIOReporter {
         currentTest.addLabel('thread', cid)
     }
 
+    getLabels(test){
+        var tags = test.tags
+        var labels = []
+        if (tags != null && tags != undefined){
+            tags.forEach(function (tag) {
```

Comment:
I would prefer arrow function here `tags.forEach((tag) => {...`. As I see in this file amount of arrow functions more than function

Reply:
Done